docs/xsl/common.xsl

```diff
@@ -54,19 +54,10 @@
     <script type='text/javascript' src='js/jquery/jquery-1.11.0.js'></script>
     <script type='text/javascript' src='js/jquery/jquery-migrate-1.2.1.min.js'></script>
 
-    <!--[if lte IE 8]>
-    <link rel="stylesheet" href="avada-stub/css/ie8.css" />
-    <![endif]-->
+    <link rel='stylesheet' id='inline-blob-janusgraph-docs-specific' href='css/docs.css' type='text/css' media='all' />
 
-    <!--[if IE]>
-    <link rel="stylesheet" href="avada-stub/css/ie.css" />
-    <![endif]-->
-
-    <link rel='stylesheet' id='style-css-css'  href='avada-stub/style.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='media-css-css'  href='avada-stub/css/media.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='animate-css-css'  href='avada-stub/css/animate-custom.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='inline-blob-from-avada-prototype' href='avada-stub/css/inlineblob.css' type='text/css' media='all' />
-    <link rel='stylesheet' id='inline-blob-from-avada-prototype-janusgraph-docs-specific' href='css/docs.css' type='text/css' media='all' />
+    <link rel='icon' type='image/jpeg' href='images/janusgraph-logomark.png' />
```

Comment:
This claims the icon is `image/jpeg` but the actual file is a PNG. On a related note, it would be just easier to put this file as `/favicon.ico` at the root of the directory, and we don't have to modify every single file or copy it into every version of every doc.